simulation_node:
  ros__parameters:
    rock_structure_mesh:
      mesh: "{{ROS2_WS}}/virtual_shake_robot_pybullet/models/SP1_PBRmodel/sp1.urdf"
      meshScale: [1.0, 1.0, 1.0]
      mass: 353.802
      restitution: 0.3
      lateralFriction: 0.5
      spinningFriction: 0.3
      rollingFriction: 0.3
      contactDamping: 100000.0
      contactStiffness: 10000000.0
      rock_position: [0.0, 0.0, 3.3]
